docs/EN_US/ECLStandardLibraryReference/SLR-Mods/DfuPlusExec.xml

```diff
@@ -23,36 +23,43 @@
         <row>
           <entry><emphasis>commandline</emphasis></entry>
 
-          <entry>A null-terminated string containing the DFUPlus.exe command
-          line to execute. The valid arguments are documented in the Client
-          Tools manual, in the section describing the DfuPlus.exe
+          <entry>A null-terminated string containing the DFUPlus command line
+          to execute. The valid arguments are documented in the Client Tools
+          manual, in the section describing the Command Line DFU
           program.</entry>
         </row>
       </tbody>
     </tgroup>
   </informaltable>
 
   <para>The <emphasis role="bold">DfuPlusExec</emphasis> action executes the
-  specified <emphasis>commandline</emphasis> just as the DfuPLus.exe program
-  would do. This simply allows you to have all the functionality of the
-  DfuPLus.exe program available within your ECL code.</para>
+  specified <emphasis>commandline</emphasis> just as the DfuPlus executable
+  program would. This allows you to have all the functionality of DfuPlus
+  available within your ECL code.</para>
+
+  <para>Unless you need to access a foreign instance of the platform, the
+  server= parameter for DfuPlus should be omitted, which then defaults to the
+  value contained in the environment's configuration. In a containerized
+  system, this defaults to the local eclservices service. If for some reason
+  that doesn't work, the default can be overridden by the value set in
+  global.defaultEsp.In a bare-metal system, this is the service named WsSMC
```

Comment:
Missing space.  Otherwise looks good.

"global.defaultEsp.In" --> "global.defaultEsp. In"

Reply:
@afishbeck I fixed the missing space.